lib/templates/document.ejs

```diff
@@ -29,6 +29,10 @@
         .message--user {
             font-weight: bold;
         }
+
+        :target {
+          background-color: yellow;
```

Comment:
This makes it so that if you click an anchor link from (e.g. from one of the comments) it will highlight the comment in yellow. We could also use a different color, yellow seemed like a decent default choice, though pure yellow is kind of bright IMO. Some other good options are gold and lightyellow.

ref: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color_keywords